App.tsx

```diff
@@ -2,18 +2,32 @@ import React from 'react';
 import {NavigationContainer} from '@react-navigation/native';
 import {RootNavigator} from './src/navigation/RootNavigator';
 import RNBootsplash from 'react-native-bootsplash';
+import {store, RootStoreContext} from './src/models';
```

Comment:
Sort the imports

Reply:
Also, Can we move App.tsx within src folder?

Reply:
ok i will fix it.